interfaces/Portal/.prettierignore

```diff
@@ -1,24 +1,5 @@
-# Make sure to keep contents aligned with all other interfaces!
-#
-# See .gitignore
-coverage/
-
-# Angular/Ionic related files
-.angular/
-www/
-dist/
-platforms/
-plugins/
-resources/
-
-
-# Exclude (generated) package files
-package*.json
-
-
 # Don't reorder imports for polyfills:
 src/polyfills.ts
 
-
 # Exception for inline code/styles
 src/index.html
```

Comment:
Indeed, this isn't being applied but it's because eslint is applying the prettier rules to js,ts,HTML files now, which doesn't look at `prettierignore`. 

I ended up just removing this file instead, because I don't think it was relevant anymore after this refactor.